### ISSUE TITLE
detect failed Loading None ...

### ISSUE THREAD

**Issue by mevius123** [thtrieu/darkflow#232]
hello,
I tried to test the model that I'd trained (with default setting) on voc2012
(2 classes /5 images per class I picked) , even the loss is close to 0 but 
still can't detect anything.

I used the following command:
`./flow --test test/ --model cfg/yolo-new.cfg 7000 --gpu 0 --json --threshold 0.1`

and I found Loading None ... 
```
Parsing cfg/yolo-new.cfg
Loading None ...
Finished in 6.69956207275e-05s
Building net ...

Source | Train? | Layer description                | Output size
-------+--------+----------------------------------+---------------
       |        | input                            | (?, 416, 416, 3)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 416, 416, 16)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 208, 208, 16)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 208, 208, 32)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 104, 104, 32)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 104, 104, 64)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 52, 52, 64)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 52, 52, 128)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 26, 26, 128)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 26, 26, 256)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 13, 13, 256)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 512)
 Load  |  Yep!  | maxp 2x2p0_1                     | (?, 13, 13, 512)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Init  |  Yep!  | conv 1x1p0_1    linear           | (?, 13, 13, 35)
-------+--------+----------------------------------+---------------
```
did I miss something?
Thanks for your help.

**Comment by EmmanouelP** [comment]
There seems to be missing the ,--load , argument in the above command.Try, --load -1, to load the checkpoint of  your training.

**Comment by mevius123** [comment]
@EmmanouelP Thanks for your reply, yeah i just forgot to type that one, but i found out after i trained a new one append this command "--trainer adam", it started detect something, so i think cause this problem is forgot to initialize the model.

Issue resolved when I add "--trainer adam" command at training.

**Comment by EmmanouelP** [comment]
Hello man
So i am in the same position as you and i try to make some training using
darkflow.My loss is also close to zero (~0.06) but when i try testing on my
images it fails to detect anything.The command i use is:

python3 ./flow --train --model cfg/tiny-yolo-new.cfg --dataset
"/home/manolis/VOC/VOC2007/JPEGImages/" --annotation
"/home/manolis/VOC/VOC2007/Annotations/" --epoch 100 --gpu 0.8

I used 2 classes and 3 images per class in orded for my training to
overfitt those images.
Could you please tell me what command did you use?
thanks in advance

2017-05-18 12:04 GMT+03:00 mevius123 <notifications@github.com>:

**Comment by EmmanouelP** [comment]
@mevius123 Hello man So i am in the same position as you and i try to make some training using darkflow.My loss is also close to zero (~0.06) but when i try testing on my images it fails to detect anything.The command i use is: 

python3 ./flow --train --model cfg/tiny-yolo-new.cfg --dataset "/home/manolis/VOC/VOC2007/JPEGImages/" --annotation "/home/manolis/VOC/VOC2007/Annotations/" --epoch 100 --gpu 0.8 

I used 2 classes and 3 images per class in orded for my training to overfitt those images. Could you please tell me what command did you use? thanks in advance
